test/integration/build/test_build.py

```diff
@@ -80,6 +81,272 @@ def test_app_without_rockspec(cartridge_cmd, project_without_dependencies):
     assert rc == 1, 'Building project should fail'
     assert 'Application directory should contain rockspec' in output
 
+    rocks_make_output = "Error: File not found: {}".format(project.rockspec_name)
```

Comment:
In fact, this is the other test - it's "bad path to rockspec passed" test.

You can simply call `... --spec bad-name` and check error.

Moreover, now this error looks not very nice:

```
Error: File not found: myapp-scm-2.rockspec
   ⨯ Failed to install rocks: Failed to run
/usr/local/bin/tarantoolctl rocks make myapp-scm-2.rockspec

exit status 1
```

I think that CLI can check file existing if `--spec` is specified and show beautiful error.